docs/intro/getting-started.rst

```diff
@@ -16,6 +16,12 @@ The first step is to create a project directory, and clone Batavia:
     $ cd tutorial
     $ git clone https://github.com/pybee/batavia.git
 
+Batavia now requires a copy of Ouroboros (the Python standard library, written in Python) to build, so we also need to clone that.
```

Comment:
A similar instruction should also be needed for the developer instructions; that section creates a completely new checkout and environment, so a fresh ouroboros checkout will also be required.